parser_library/src/checking/asm_instr_check.cpp

```diff
@@ -833,82 +833,6 @@ bool entry::check(const std::vector<const asm_operand*>& to_check,
     return true;
 }
 
-end::end(const std::vector<label_types>& allowed_types, const std::string& name_of_instruction)
```

Comment:
Why move the checking of the instruction out of the checker? It is more complicated to check it in asm_processor, since there are some simplifying layers in between...

Reply:
Yes, I am moving it back. Keeping only the new check